windows-iotcore/tutorials/quickstarter/PrototypeBoards.md

```diff
@@ -24,11 +24,11 @@ Below you'll find boards we suggest to help you get started with Windows 10 IoT
 |-----------|----------|---------|---------|---------|---------|-------|
```

Comment:
Sydney, I am not sure what happened here but the table rendering is broken.  There are not the same number of columns in the header row as there is in the second row causing the table not to render as a table.  Please verify the header and second row match the number of columns in the rest of the table.